extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -85,6 +85,7 @@ function DefaultArrow({by, ...rest}) {
   const classes = useStyles();
   return (
     <button
+      part={`arrow ${by > 0 ? 'next' : 'prev'}`}
```

Comment:
This is noop in light dom?

Reply:
Correct. Doesn't do anything in the light dom.